pyproject.toml

```
@@ -94,8 +94,8 @@ Documentation = "https://napari.org"
 pyside2 = [
     "PySide2>=5.15.1 ; python_version < '3.11' and platform_machine != 'arm64'",
 ]
-pyside6_experimental = [
-    "PySide6 < 6.5 ; python_version < '3.12'"
+pyside6 = [
+    "PySide6"
```

```suggestion
    "PySide6 > 6.7"
```
Like pyqt6, versions before 6.5 have a flickering issue (https://github.com/napari/napari/issues/5644) and before 6.7 TypeError
(https://github.com/napari/napari/issues/5703)